### DISCUSSION TITLE
Drag and drop File names into text.

### DISCUSSION THREAD

**Discussion by stovedoctor** [HASwitchPlate/openHASP#495] [Documentation]
Wow Just playing around found a Easter Egg sort of....
You can drag and drop files as the name and directory into your text file and it adds the / before the file name. 
Just let go of mouse at the position in your text file you want file name to drop. 

https://github.com/HASwitchPlate/openHASP/assets/60519485/4d0757f6-2e73-44b9-be47-413b352e510c

**Comment by fvanroie** [comment]
The idea is to be able to drag files into directories to move them... but this features isn't implemented in the front-end and back-end yet.
